app/controllers/books/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Books
+  class CommentsController < ApplicationController
+    def create
+      book = Book.find(params[:book_id])
+      comment = book.comments.new(comments_params)
+      comment.save
+      flash[:invalid_comment] = comment.errors.full_messages.to_sentence
+      redirect_to book_path(book)
+    end
+
+    def destroy
+      book = Book.find(params[:book_id])
+      comment = book.comments.find(params[:id])
+      comment.destroy!
+      redirect_to book_path(book)
+    end
+
+    private
+
+    def comments_params
+      { **params.require(:comment).permit(:body), user: current_user }
```

Comment:
```suggestion
      { user: current_user, **params.require(:comment).permit(:body) }
```

好みの問題ですが、`*`や`**`は最後に持ってくることが多い気がしますー。